File: blingbling/src/main/java/dinson/customview/_global/ConstantsUtils.java
package dinson.customview._global;

import android.os.Environment;

import java.io.File;

import dinson.customview.utils.UIUtils;

/**
 * 常量类
 */
public class ConstantsUtils {
    public static String SDCARD_PRIVATE = UIUtils.getContext().getExternalCacheDirs()[0].getPath();
    public static String SDCARD_PRIVATE_CACHE = UIUtils.getContext().getExternalFilesDirs("cache")[0].getPath() + File.separator;
    public static String SDCARD_PRIVATE_IMAGE = UIUtils.getContext().getExternalFilesDirs("Panorama")[0].getPath() + File.separator;


    public static String SDCARD_ROOT = Environment.getExternalStorageDirectory() + File.separator;

    public static String LOGCAT_TAG = "│ --> ";
    public static String PACKAGE_NAME = "dinson.customview";
    public static String WANANDROID_DOMAIN = "www.wanandroid.com";
}

File: blingbling/src/main/java/dinson/customview/_global/ConstantsUtils.kt
package dinson.customview._global

import android.os.Environment
import com.dinson.blingbase.RxBling

import java.io.File

/**
 * 常量类
 */
object ConstantsUtils {
    val SDCARD_PRIVATE = RxBling.getApplicationContext().externalCacheDirs[0].path + File.separator
    val SDCARD_PRIVATE_CACHE = RxBling.getApplicationContext().getExternalFilesDirs("cache")[0].path + File.separator
    val SDCARD_PRIVATE_IMAGE = RxBling.getApplicationContext().getExternalFilesDirs("Panorama")[0].path + File.separator

    val SDCARD_ROOT = Environment.getExternalStorageDirectory().toString() + File.separator

    const val LOGCAT_TAG = "│ --> "
    const val WAN_ANDROID_DOMAIN = "www.wanandroid.com"

    const val APP_FONT_PATH = "fonts/AppleFontWithoutChinese.ttf"
    const val ICON_FONT_PATH = "iconfont/iconfont.ttf"
}
